packages/gatsby-cli/src/reporter/redux/index.ts

```diff
@@ -11,6 +13,75 @@ let store = createStore(
   {}
 )
 
+function calculateTimeoutDelay(
+  envVarValue: string | undefined,
+  defaultValue: number,
+  min: number
+): number | null {
+  if (!envVarValue) {
+    return null
```

Comment:
I would return `0` instead to keep the type `number`.

Reply:
Done as part of https://github.com/gatsbyjs/gatsby/pull/26618/commits/e10a55644f3f0dd8a4bd419c7fa23075660e3e93